dex/testing/dash/harness.sh

```diff
@@ -0,0 +1,370 @@
+sleep 3
+
+################################################################################
+# Create the gamma wallet
+################################################################################
+echo "Creating gamma wallet - encrypted HD"
+
+tmux send-keys -t $SESSION:2 "./alpha createwallet \"gamma\" false false \"${WALLET_PASSWORD}\" ${DONE}" C-m\; ${WAIT}
+sleep 2
+
+tmux send-keys -t $SESSION:2 "./alpha -rpcwallet=\"gamma\" upgradetohd \"${GAMMA_SEED_MNEMONIC}\" \"\" \"${WALLET_PASSWORD}\"${DONE}" C-m\; ${WAIT}
+sleep 3
+
+################################################################################
+# Create the delta wallet
+################################################################################
+echo "Creating delta wallet - encrypted HD"
+
+tmux send-keys -t $SESSION:2 "./beta createwallet \"delta\" false false \"${WALLET_PASSWORD}\" ${DONE}" C-m\; ${WAIT}
+sleep 3
+
+tmux send-keys -t $SESSION:2 "./beta -rpcwallet=\"delta\" upgradetohd \"${DELTA_SEED_MNEMONIC}\" \"\" \"${WALLET_PASSWORD}\"${DONE}" C-m\; ${WAIT}
+sleep 3
+
+################################################################################
+# Mine alpha
+################################################################################
+echo "Generating 200 blocks for alpha"
+
+tmux send-keys -t $SESSION:2 "./alpha generatetoaddress 100 ${ALPHA_MINING_ADDR}${DONE}" C-m\; ${WAIT}
+sleep 1
+tmux send-keys -t $SESSION:2 "./alpha generatetoaddress 100 ${ALPHA_MINING_ADDR}${DONE}" C-m\; ${WAIT}
+sleep 1
+
+################################################################################
+# Send gamma and delta some coin
+################################################################################
+echo "Funding beta, gamma & delta wallets from alpha's block subsidy. 500 tDASH/block."
+
+tmux send-keys -t $SESSION:2 "./alpha walletpassphrase \"${WALLET_PASSWORD}\" 100000000${i}${DONE}" C-m\; ${WAIT}
+
+#  Send some dash to beta, gamma and delta wallets, mining a few more blocks.
+echo "Sending 8777 DASH each to beta, gamma and delta in 8 blocks"
+for i in 1000 1800 500 700 100 1500 300 2500 200 100 7 7 7 7 7 7 7 3 3 3 3 3 3 2 2 2 2 2 1
+do
+  tmux send-keys -t $SESSION:2 "./alpha sendtoaddress ${BETA_MINING_ADDR} ${i}${DONE}" C-m\; ${WAIT}
+  tmux send-keys -t $SESSION:2 "./alpha sendtoaddress ${GAMMA_ADDRESS} ${i}${DONE}" C-m\; ${WAIT}
+  tmux send-keys -t $SESSION:2 "./alpha sendtoaddress ${DELTA_ADDRESS} ${i}${DONE}" C-m\; ${WAIT}
+  #
+  tmux send-keys -t $SESSION:2 "./mine-beta 1${DONE}" C-m\; ${WAIT}
+done
+
+################################################################################
+# Setup watch background miner -- if required
+################################################################################
+if [ -z "$NOMINER" ] ; then
+  tmux new-window -t $SESSION:3 -n "miner" $SHELL
+  tmux send-keys -t $SESSION:3 "cd ${HARNESS_DIR}" C-m
+  tmux send-keys -t $SESSION:3 "watch -n 15 ./mine-alpha 1" C-m
+fi
+
```

Comment:
Can we add a miner?

Reply:
![image](https://github.com/decred/dcrdex/assets/126670673/5e230d9d-8df3-4966-9831-e12613495e58)

Added